frontend/src/App.css

```diff
@@ -3,10 +3,14 @@
 }
 
 #applicationRootDiv {
-  margin-bottom: var(--footerHeight);
+  margin-bottom: 10vh;
```

Comment:
Wofür die margin bootm?

Reply:
der war vorher da, aber zu groß, daher hab wir das nochmal bissl kleiner gemacht, das is der abstand zwischen footer und content